torch/_inductor/codecache.py

```diff
@@ -975,7 +979,10 @@ def get_glibcxx_abi_build_flags() -> str:
 
 
 def cpp_flags() -> str:
-    return "-std=c++17 -Wno-unused-variable -Wno-unknown-pragmas"
+    flags = ["-std=c++17", "-Wno-unused-variable", "-Wno-unknown-pragmas"]
+    if is_clang():
+        flags.append("-Werror=ignored-optimization-argument")
```

Comment:
The switch is only available with internal clang. Is this flags change for both internal and external?

Reply:
Yes, this option is applicable to both internal and external clang compilers. I think it's fine, because we want to get long-compilation-time signals from non-fb models as well.